versions.gradle

```diff
@@ -11,7 +11,7 @@ ext {
         androidSupport: '25.3.1',
         androidSupportTest: '0.5',
         bullet: 'e5044b53df',
-        butterKnife: '8.2.1',
+        butterKnife: '8.5.1',
         crashlytics: '2.6.7@aar',
         dagger: '2.6',
```

Comment:
Jared needs an update and it was just released? [cantkeepup](https://images.bigcartel.com/product_images/170534512/CANT_KEEP_BLK.jpg)